### PR TITLE
feat: Org member personal contribution and active rank set up

### PR DESCRIPTION
**Pull request by shuofan**
#### What this PR does / why we need it:
Org member personal contribution and active rank set up

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @Effet 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |  Personal contribution and active rank  |
| 🇨🇳 中文    |    个人贡献和活跃排行          |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by sfwn — DISMISSED** [review]

File: .erda/migrations/cmdb/20211229-member-active-rank.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    `updated_at` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT 'updated time',
+    `org_id` varchar(36) NOT NULL COMMENT 'org id',
+    `user_id` varchar(36) NOT NULL COMMENT 'user id',
+    `issue_score` bigint(20) NOT NULL DEFAULT '0' COMMENT 'issue',
+    `commit_score` bigint(20) NOT NULL DEFAULT '0' COMMENT 'commit',
+    `quality_score` bigint(20) NOT NULL DEFAULT '0' COMMENT 'quality',
+    `total_score` bigint(20) NOT NULL DEFAULT '0' COMMENT 'total',
+    `soft_deleted_at` bigint(20) NOT NULL DEFAULT 0 COMMENT 'deleted at',
+    PRIMARY KEY (`id`),
+    KEY `idx_org_id_user_id` (`org_id`, `user_id`)
```
Comment (Effet) [inline .erda/migrations/cmdb/20211229-member-active-rank.sql:13]: i think it should be a unique key, and key of (soft_deleted_at, org_id, user_id)

**Review by Effet — CHANGES_REQUESTED** [review]

File: .erda/migrations/cmdb/20211229-member-active-rank.sql (hunk 1 above)
Comment (shuofan) [reply .erda/migrations/cmdb/20211229-member-active-rank.sql]: done

**Review by Effet — APPROVED** [review]

**Pull request merged**